tox.ini

```diff
@@ -31,12 +32,14 @@ deps =
     drf39: djangorestframework>=3.9,<3.10
     drf310: djangorestframework>=3.10,<3.11
     drf311: djangorestframework>=3.11,<3.12
+    pyjwt1: PyJWT[crypto]>=1.5.2,<2.0.0
+    pyjwt2: PyJWT[crypto]>=2.0.0,<3.0.0
     coverage: coverage
     codecov: codecov
 # bash is used to create the codecov flags from the envname,
 # only when codecov is used in the envname
 whitelist_externals =
-    /bin/bash,
+    /bin/bash
```

Comment:
Not directly part of this, but fixes the annoying `WARNING: test command found but not installed in testenv` log lines